pom.xml

```diff
@@ -118,6 +118,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
```

Comment:
This needs to be here as it generates a meta-data file that will be packaged it into the jar of the library. As an example, this enables auto-completion in the properties in IntelliJ Idea.